### PR TITLE
Converted fish bucket dispense behaviors to non overriding system

### PR DESCRIPTION
**Pull request by abigailfails**
Minor change, essentially just replacing the `FishBucketDispenseBehavior` that was used as a fallback with whatever the previous behavior was at registry time, increasing compatibility with other mods.

### CODE REVIEW

File: src/main/resources/META-INF/mods.toml [hunk 1]
```diff
@@ -31,6 +31,6 @@ license = "https://github.com/team-abnormals/upgrade-aquatic/blob/main/LICENSE.t
 [[dependencies.upgrade_aquatic]]
     modId = "abnormals_core"
     mandatory = true
-    versionRange = "[3.0.4,)"
+    versionRange = "[3.0.6,)"
```
Comment (SmellyModder) [inline src/main/resources/META-INF/mods.toml:34]: Not sure why this is included?

**Review by SmellyModder — CHANGES_REQUESTED** [review]

File: src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java [hunk 2]
```diff
@@ -128,8 +129,8 @@ protected ItemStack dispenseStack(IBlockSource source, ItemStack stack) {
                     }
                 }
             }
-            return new FishBucketDispenseBehavior().dispense(source, stack);
-        }
+            return stack; //Should never be reached
+		}
```
[the remark this answers is not in the document]
Comment (abigailfails) [reply src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java:133]: Yeah, fair enough. It's probably clear enough from the predicate that it's a fallback

File: src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java [hunk 3]
```diff
@@ -128,8 +129,8 @@ protected ItemStack dispenseStack(IBlockSource source, ItemStack stack) {
                     }
                 }
             }
-            return new FishBucketDispenseBehavior().dispense(source, stack);
-        }
+            return stack;
+	}
```
Comment (SmellyModder) [inline src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java:133]: The formatting got a bit messed up here.

**Review by SmellyModder — CHANGES_REQUESTED** [review]

File: src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java (hunk 3 above)
Comment (abigailfails) [reply src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java:133]: Hmm yeah, that might be an intellij moment. I'll try to fix it.

Comment (abigailfails) [reply src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java]: Wait no it looks like that was just a me moment. In any case, it's fixed (I think).

File: src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java [hunk 4]
```diff
@@ -128,8 +129,8 @@ protected ItemStack dispenseStack(IBlockSource source, ItemStack stack) {
                     }
                 }
             }
-            return new FishBucketDispenseBehavior().dispense(source, stack);
-        }
+            return stack;
+		}
```
Comment (SmellyModder) [inline src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java:133]: Everything else seems good.
This bracket still looks to be formatted incorrectly.

Comment (abigailfails) [reply src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java:133]: Oh god it's still broken, hang on

File: src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java (hunk 3 above)
Comment (abigailfails) [reply src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java]: Ok sorry about the wait i ended up getting swamped by school, i think it's fixed now - the changed files tab shows no difference in the indentation after the commit i just made

File: src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java (hunk 4 above)
Comment (abigailfails) [reply src/main/java/com/minecraftabnormals/upgrade_aquatic/core/other/UADispenseBehaviorRegistry.java]: might not be broken now?

**Review by SmellyModder — APPROVED** [review]

**Pull request merged**